lib/inferno/dsl/auth_info.rb

```diff
@@ -161,13 +161,12 @@ def to_s
 
       # @private
       def add_to_client(client)
-        # TODO
-        # client.auth = self
-        # self.client = client
+        client.auth_info = self
+        self.client = client
+        # TODO: do we want to perform authorization if no access_token or rely on SMART/ other auth tests?
```

Comment:
This will have to wait. Performing auth (other than backend services) will require setting up redirect endpoints in core and will be a larger undertaking.